Makefile

```diff
@@ -219,6 +224,15 @@ test-back: ## run back-end tests
 	@DB_PORT=$(DB_PORT) bin/pytest
 .PHONY: test-back
 
+# -- Cookie cutter
+
+start-project: ## bootstrap a new project based on Richie using Cookiecutter
+	@$(COMPOSE_RUN_COOKIECUTTER) \
+	    "/app/$$(basename $(PWD))/cookiecutter" \
+	    -o /app \
+		--config-file "/app/$$(basename $(PWD))/.cookiecutterrc"
+.PHONY: start-project
```

Comment:
Should be tested in the CI. WDYT?

Reply:
Yes I will open an issue for this as this PR is already too big.